gravitee-apim-repository/gravitee-apim-repository-elasticsearch/src/test/java/io/gravitee/repository/elasticsearch/analytics/ElasticsearchAnalyticsRepositoryTest.java

```diff
@@ -192,7 +192,7 @@ public void testCountWithTwoApis() throws Exception {
         );
 
         assertThat(response).isNotNull();
-        assertThat(response.getCount()).isEqualTo(4);
+        assertThat(response.getCount()).isEqualTo(7);
```

Comment:
Why did you need to change this test? I don't see the correspondance between the content of the PR and this test.

Reply:
It was a flaky test, to be honest. Not sure why but sometimes it passes with 4, sometimes passes with 7.